azurerm/internal/services/azurestackhci/tests/hci_cluster_resource_test.go

```diff
@@ -0,0 +1,228 @@
+package tests
```

Comment:
Can we convert this to the latest test code style?

Reply:
Could you share me an example? Thanks.